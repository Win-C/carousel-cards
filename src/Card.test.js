import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it("smoke test - renders without crashing", function () {
  render(
    <Card
      caption="captionTest"
      src="imageTest"
      currNum="1"
      totalNum="1"
    />
  );
});

it("snapshot test - matches snapshot", function () {
  const { container } = render(
    <Card
      caption="captionTest"
      src="imageTest"
      currNum="1"
      totalNum="1"
    />
  );
  expect(container).toMatchSnapshot();
});
