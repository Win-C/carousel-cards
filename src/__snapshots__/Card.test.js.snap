// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test - matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      captionTest
    </h4>
    <img
      alt="captionTest"
      class="Card-image"
      src="imageTest"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      1
      .
    </small>
  </div>
</div>
`;
